packages/backend/src/routes/CommentRouter.ts

```diff
@@ -38,7 +40,9 @@ async function loadComment(req, res, next) {
             _id: req.params.id,
         },
     })
-    res.json(comment)
+    if (!comment || comment.content === DELETED_CONTENT)
+        res.status(404).json('no such comment')
+    else res.json(comment)
```

Comment:
I think the deleted comment should be shown in frontend as `DELETED_CONTENT`
Since the upvote/downvote are still exiting
we should let users know where their Reps from